src/test/java/easytests/core/mappers/TopicsMapperTest.java

```diff
@@ -15,84 +15,94 @@
  * @author malinink
  */
 public class TopicsMapperTest extends AbstractMapperTest {
+
+    private TopicsSupport topicsSupport = new TopicsSupport();
+
     @Autowired
     private TopicsMapper topicsMapper;
 
     @Test
     public void testFindAll() throws Exception {
-        final List<TopicEntity> topicsEntities = this.topicsMapper.findAll();
-        Assert.assertEquals((Integer) 3, (Integer) topicsEntities.size());
+        final List<TopicEntity> topicEntities = this.topicsMapper.findAll();
+
+        Assert.assertEquals(3, topicEntities.size());
+
+        Integer index = 0;
+        for(TopicEntity topicEntity: topicEntities) {
+
+            this.topicsSupport.assertEquals(this.topicsSupport.getEntityFixtureMock(index), topicEntity);
+
+            index++;
+        }
     }
 
     @Test
     public void testFind() throws Exception {
-        final TopicEntity topicEntity = this.topicsMapper.find(1);
-        Assert.assertEquals((Integer) 1, topicEntity.getId());
-        Assert.assertEquals("Name1", topicEntity.getName());
-        Assert.assertEquals((Integer) 2, topicEntity.getSubjectId());
+        final TopicEntity topicFixtureEntity = this.topicsSupport.getEntityFixtureMock(0);
+        final TopicEntity topicEntity = this.topicsMapper.find(topicFixtureEntity.getId());
+
+        this.topicsSupport.assertEquals(topicFixtureEntity, topicEntity);
     }
 
     @Test
     public void testFindBySubjectId() throws Exception {
```

Comment:
You should not check cases that cover all fixtures here, just check one of them, that returns list of more than one item.
For asserting you could prepare list manually from FixtureMocks.

Also you should check that find for absent subject id response list with no items, but that must be another test `testFindBySubjectIdWithoutTopics`

Reply:
Look the same in `testFindByUserIdWithoutSubjects` `testFindByUserId`

Reply:
@malinink 
Done. But, though, why didn't you create the same test (testFindByEmailWithoutUsers) in your UsersMapper class test which is an "illustration" of what we should do in tests of the mapper type.

Reply:
Hm, It is a good question! Should we test empty results or not? Could you research that?

Reply:
@malinink 
Why can't we just pass an invalid emai parameter into 'findByEmail()' and then check if it would be 'null'?